docs/getting-setup.md

```diff
@@ -73,11 +73,11 @@ This command starts a [development server][dev-server].
 
 ### Step 5. Open the Debugger
 
-Go to `localhost:8000` in any browser to view the Debugger. If everything worked successfully, you should see this [screenshot](https://cloud.githubusercontent.com/assets/254562/20439428/7498808a-ad89-11e6-895d-d6db320c5009.png)
+Go to http://localhost:8000 in any browser to view the Debugger. If everything worked successfully, you should see something like this [screenshot](https://cloud.githubusercontent.com/assets/2134/22162697/913777b2-df04-11e6-9150-f6ad676c31ef.png) :sweat_smile:
```

Comment:
I wasn't sure about the screenshots, there's no change to use them anywhere else.  I can move them if you feel strongly, I just wasn't sure.